mu_repo/action_clone.py

```diff
@@ -0,0 +1,127 @@
+from mu_repo.execute_command import ExecuteCommand
+import os
+
+#===================================================================================================
+# Run
+#===================================================================================================
+def Run(params):
+    '''
+    mu-repo can deal with cloning a repository and other referenced repositories recursively.
+
+    To work this way, users are expected to commit the .mu_repo files they rely on and set
+    the base urls that users work with.
+
+    i.e.:
+
+    Say that someone is working with 3 projects: A, B and C and C depends on B and B depends on A
+    and they're all available on https://github.com/myuser
+
+    The urls in this case to be checked out would be something as:
+
+    ssh://git@github.com:myuser/projectA.git
+
+    https://github.com/myuser/projectA.git
+
+    So, the url: https://github.com/myuser or git@github.com:myuser has to be configured as the
+    'base' url for clones.
+
+    mu-repo in this case will work with git to provide that configuration, so, one should first
+    add that remote for mu-repo to work with:
+
+    git config --global --add mu-repo.remote-host ssh://git@github.com:myuser
+
+    Note that it's possible to add as many urls as wanted (they'll all be checked later on
+    when cloning the project). Note that multiple remote hosts may be added that way.
```

Comment:
The line with the `Note` is redundant. I think it would be better to get rid of it and mention that the remotes will be tried in the order they were added.